core/unit_test/CMakeLists.txt

```diff
@@ -25,6 +25,24 @@ TARGET_INCLUDE_DIRECTORIES(kokkos_gtest PUBLIC ${GTEST_SOURCE_DIR})
 #Gtest minimally requires C++11
 TARGET_COMPILE_FEATURES(kokkos_gtest PUBLIC cxx_std_11)
 
+#
+# Define Incremental Testing Feature Levels
+# Define Device name mappings (i.e. what comes after Kokkos:: for the ExecSpace)
+#
+
+SET(KOKKOS_CUDA_FEATURE_LEVEL 999)
+SET(KOKKOS_CUDA_NAME Cuda)
+SET(KOKKOS_HIP_FEATURE_LEVEL 5)
+SET(KOKKOS_HIP_NAME Experimental::HIP)
+SET(KOKKOS_OPENMP_FEATURE_LEVEL 999)
+SET(KOKKOS_OPENMP_NAME OpenMP)
+SET(KOKKOS_OPENMPTARGET_FEATURE_LEVEL 10)
+SET(KOKKOS_OPENMPTARGET_NAME Experimental::OpenMPTarget)
+SET(KOKKOS_SERIAL_FEATURE_LEVEL 999)
+SET(KOKKOS_SERIAL_NAME Serial)
+SET(KOKKOS_THREADS_FEATURE_LEVEL 999)
+SET(KOKKOS_THREADS_NAME Threads)
```

Comment:
We may want to move this in a more general place later (the KOKKOS_*_NAME thing since it is generally useful)